### PR TITLE
Fix #3676 [Frontend] Fix the twitter feed loading when user navigates back to homepage

### LINKED ISSUE #3676: [Frontend] Fix the twitter feed loading when user navigates back to homepage
### Description

The twitter feed homepage fails to load when a users visits some other page after visiting the homepage and then comes back to the homepage. Steps to reproduce:

1. Open EvalAI homepage on FrontendV2
2. Navigate to login/signup page
3. Navigate back to homepage and check the twitter feed

Screenshot for reference:

Screenshot from 2022-01-17 18-02-21

Expected behavior:

Screenshot from 2022-01-17 18-03-28

### PR DESCRIPTION
**Pull request by haldendmello**
**Description**

The twitter feed homepage use to fails when a users visits some other page after visiting the homepage and then comes back to the homepage. 

**Steps to reproduce:**

Open EvalAI homepage on FrontendV2
Navigate to login/signup page
Navigate back to homepage and check the twitter feed

**GIF for reference:**
twitterFeedIssue

### CODE REVIEW

**Review by mr-vaibh — APPROVED** [review]

File: frontend_v2/src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -94,9 +94,11 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
 
   /**
    * Set SUBSCRIBE_FORM to this.components after view initialization
+   * (<any>window).twttr.widgets.load() this load Twitter Feed
```
Comment (Ram81) [inline frontend_v2/src/app/components/home/home.component.ts:97]: @haldendmello can you remove this comment? Rest of the PR LGTM

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: frontend_v2/src/app/components/home/home.component.ts (hunk 1 above)
Comment (haldendmello) [reply frontend_v2/src/app/components/home/home.component.ts:97]: ok

Comment (haldendmello) [reply frontend_v2/src/app/components/home/home.component.ts]: @Ram81 I have removed the comment.

**Review by Ram81 — APPROVED** [review]
LGTM

@RishabhJain2018 this can be merged

**Pull request merged**